client/src/components/EditFeedback/index.js

```diff
@@ -82,14 +80,7 @@ const EditFeedback = ({ feedbackToBeEdited, noShowPage, updatePreviousFeedback,
 							placeholder={feedbackToBeEdited.text}
 						></textarea>
 					</div>
-					<div>
-						<input
-							className="feedback-time"
-							type="text"
-							name="time"
-							value={feedbackToBeEdited.time}
-						/>
-					</div>
+
 					<div>
 						<input
```

Comment:
No need to edit mentor's name. It should keep the same original name.
Also, we will try to get the name from login without inputting it.

Reply:
Until login is in place we need mentor be able to amend the name. They may mistype it as it is free text. Once login is available, I can easily remove it.